### PR TITLE
 please add blockton blockchain to chainlist

### PR DESCRIPTION
**Pull request by blocktoncoin**
please add blockton blockchain to chainlist

### CODE REVIEW

File: _data/chains/eip155-8272.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://blocktoncoin.com",
+  "shortName": "BTON",
+  "chainId": 8272,
+  "networkId": 8272,
+  "explorers": [
+    {
+      "name": "Blockton Explorer",
+      "url": "https://blocktonscan.com/",
```
Comment (ligi) [inline _data/chains/eip155-8272.json:19]:
```suggestion
      "url": "https://blocktonscan.com",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by ligi** [comment]
The contrast in your explorer is really bad:

image

**Comment by blocktoncoin** [comment]
yes ligi the transaction is ongoing.

**Comment by blocktoncoin** [comment]
please add the chain

**Review by ligi — APPROVED** [review]

**Comment by blocktoncoin** [comment]
thanks @ligi for adding blockton when we expect blockton live on chainlist.org

**Comment by blocktoncoin** [comment]
hello please add our mainnet coin we are listed on cmc

https://coinmarketcap.com/currencies/blockton/

**Pull request merged**